rskj-core/src/main/java/co/rsk/rpc/json/JsonResponseSizeLimiter.java

```diff
@@ -0,0 +1,106 @@
+package co.rsk.rpc.json;
+
+import co.rsk.rpc.exception.JsonRpcResponseLimitException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.ethereum.rpc.exception.RskJsonRpcRequestException;
+
+import java.util.Iterator;
+import java.util.Map;
+
+public class JsonResponseSizeLimiter {
+    private static final int BRACKETS_SIZE_IN_BYTES = 2;
+    private static final int COMMA_SIZE = 1;
+    private static final int JSON_FILED_MISSING_SYMBOLS = 3;
+    private static final long serialVersionUID = -4364188205571356867L;
+
+    private int acc = 0;
```

Comment:
I feel changing this variable in multiple instanced methods is not a good idea. Not sure if sharing the state is a good idea.

I could be wrong.

What do you think @Vovchyk  @illuque-iov ?

Reply:
I'm not sure if I got your point. 
This value is not static and each object has its own value to sum the total length in bytes. From other parts of the code we call the static method, getSizeInBytesWithLimit and this method is the one in charge of instantiate the object and return the result.

Reply:
Probably Reynold's doubt can be avoided if you create a explicit creator method outside of `getSizeInBytesWithLimit` and then invoke `getSizeInBytesWithLimit` from it.